lib/core/cloud_events_schemas/cloud_event_v_1_0_1.ex

```diff
@@ -0,0 +1,107 @@
+# credo:disable-for-next-line
+defmodule Polyn.CloudEvent.V_1_0_1 do
```

Comment:
Is there a reason to go this route over loading the schema from a file?

Reply:
Good question. It would be nicer to just keep in a file. I'll give that a shot.

Reply:
@fugufish That worked better. Good call out. Updated in https://github.com/SpiffInc/polyn-elixir/pull/1/commits/328c628d7cc8fa7a03b32f0b18f35f86cb6b0ba7